AdvCore_GridCompMod.F90

```diff
@@ -539,7 +540,12 @@ subroutine Run(GC, IMPORT, EXPORT, CLOCK, RC)
       VERIFY_(STATUS)
       AllOCATE( BK_r8(LM+1) ,stat=STATUS )
       VERIFY_(STATUS)
-      call set_eta(LM,LS,ptop_r8,pint_r8,ak_r8,bk_r8)
+      call MAPL_GetResource(MAPL, eta_rc_file, label='ETA_RC_FILE:', default = 'None', rc = status)
```

Comment:
It is very odd that this is in the run() method rather than the init() method.    We need to talk to Atanas about this, and maybe Bill.

We also may need to consider adding an MPI broadcast, and only reading on a single process.  I had not anticipated that aspect.

Reply:
It is moved. But do wee need the broadcast? That can be done in MAPL_EtaVerticalCorrdinate

Reply:
I agree it could be done in the EtaVerticalCoordinate implementation, but ... we'd have to rely on getting the current VM.   I prefer to avoid such things.   (Feels a bit too much like using MPI comm world.)

So long as it is in the init, the performance is probably fine.   We can always broadcast at a later date.